Tags/Tag.js

```diff
@@ -4,8 +4,8 @@ import { View, Text, TextInput, TouchableOpacity } from "react-native";
 
 import styles from "./styles";
 
-const Tag = ({ label, onPress, tagContainerStyle, tagTextStyle }) => (
-  <TouchableOpacity style={[styles.tag, tagContainerStyle]} onPress={onPress}>
+const Tag = ({ label, onPress, tagContainerStyle, tagTextStyle, readonly }) => (
+  <TouchableOpacity style={[styles.tag, tagContainerStyle]} onPress={onPress} activeOpacity={readonly ? 1 : 0.2 }>
```

Comment:
Maybe you could not render the TouchableOpacity Instead?
```
Tag = (/*...*/) => {

 const TextPart = <Text style={[styles.tagLabel, tagTextStyle]}>{label}</Text>

if (readonly) {
return TextPart;
}

return (
    <TouchableOpacity style={[styles.tag, tagContainerStyle]} onPress={onPress}>
    {TextPart}
</TouchableOpacity>);

}

Reply:
Yup good call - added a conditional to switch between a View and TouchableOpacity